novuRajan/gannt-chart
# Feature to add extra info to tooltip

## Description of the problem this PR solves
We can add tooltip content 

## Details of how the solution has been implemented
tooltip was added in  the chart config and the content added in the tooltip was shown

## Testing Instructions
 ```
GanttChart.createGanttChart(data,{
        containerId: 'chart',
        tooltip: ['type'],
    });
    
```

go to the gantt chart and hover you will see the type if type is present in that task

## Browsers tested
Select all browsers the solution was tested on <br>
*(test on as many as possible if the solution is front-end, otherwise this section is optional)*

## Notes
Any further information required for this PR to be reviewed and tested
